sql/migrations/20170703175424_recreateFullBlockListView.sql

```diff
@@ -0,0 +1,63 @@
+BEGIN; 
+
+DROP VIEW IF EXISTS full_blocks_list;
+
+CREATE VIEW full_blocks_list AS
+
+SELECT b."id" AS "b_id",
+       b."version" AS "b_version",
+       b."timestamp" AS "b_timestamp",
+       b."height" AS "b_height",
+       b."previousBlock" AS "b_previousBlock",
+       b."numberOfTransactions" AS "b_numberOfTransactions",
+       (b."totalAmount")::bigint AS "b_totalAmount",
+       (b."totalFee")::bigint AS "b_totalFee",
+       (b."reward")::bigint AS "b_reward",
+       b."payloadLength" AS "b_payloadLength",
+       ENCODE(b."payloadHash", 'hex') AS "b_payloadHash",
+       ENCODE(b."generatorPublicKey", 'hex') AS "b_generatorPublicKey",
+       ENCODE(b."blockSignature", 'hex') AS "b_blockSignature",
+       t."id" AS "t_id",
+       t."rowId" AS "t_rowId",
+       t."type" AS "t_type",
+       t."timestamp" AS "t_timestamp",
+       ENCODE(t."senderPublicKey", 'hex') AS "t_senderPublicKey",
+       t."senderId" AS "t_senderId",
+       t."recipientId" AS "t_recipientId",
+       (t."amount")::bigint AS "t_amount",
+       (t."fee")::bigint AS "t_fee",
+       ENCODE(t."signature", 'hex') AS "t_signature",
+       ENCODE(t."signSignature", 'hex') AS "t_signSignature",
+       ENCODE(s."publicKey", 'hex') AS "s_publicKey",
+       d."username" AS "d_username",
+       v."votes" AS "v_votes",
+       m."min" AS "m_min",
+       m."lifetime" AS "m_lifetime",
+       m."keysgroup" AS "m_keysgroup",
+       dapp."name" AS "dapp_name",
+       dapp."description" AS "dapp_description",
+       dapp."tags" AS "dapp_tags",
+       dapp."type" AS "dapp_type",
+       dapp."link" AS "dapp_link",
+       dapp."category" AS "dapp_category",
+       dapp."icon" AS "dapp_icon",
+       it."dappId" AS "in_dappId",
+       ot."dappId" AS "ot_dappId",
+       ot."outTransactionId" AS "ot_outTransactionId",
+       ENCODE(t."requesterPublicKey", 'hex') AS "t_requesterPublicKey",
+			 CONVERT_FROM(tf."data", 'utf8') AS "tf_data",
```

Comment:
I hate these issues :P My editor shows it as correctly indented. We should consider moving to space based indentation.